keyboards/clawsome/coupe/readme.md

```diff
@@ -0,0 +1,13 @@
+# Coupe
+
```

Comment:
I like having at least a link to a picture about the board.  
Can help tremendously while helping others. (so preferably from a source that will be available long-term)

Reply:
I can link to the shop page from my website - should I add the URL there to line 2?

Reply:
Add a line like this:

`![Coupe](http://i.imgur.com/picture_of_a_coupe.jpg)`

Reply:
Added to line #3